qp_deblur/__init__.py

```diff
@@ -0,0 +1,58 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_client import QiitaPlugin, QiitaCommand
+
+from .deblur import deblur
+
+__all__ = ['deblur']
+
+
+# Initialize the plugin
+plugin = QiitaPlugin(
+    'deblur', '0.1.0', 'A greedy deconvolution algorithm based on Illumina '
+    'Miseq/Hiseq error profiles')
+
+# Define the HUMAnN2 command
+req_params = {'seqs-fp': ('artifact', ['Demultiplexed'])}
+opt_params = {
+    # parameters not being passed
+    # output-dir
+    # keep-tmp-files
+    # log-level
+    # log-file
+    # is-worker-thread
+    'ref-fp': ['choice:[""]', ''],
+    'ref-db-fp': ['choice:[""]', ''],
```

Comment:
Correct, but the users don't need to provide this parameter. If you expose this parameter here, the users can choose different values, setting the above and this one out of sync. This is "an internal" parameter for the plugin. Makes sense? You only need to expose here the ones that the user provide. Note that there is a similar situation in OTU picking (the exact same parameter is there), so it is not exposed to the user since it is used internally for optimization purposes.